src/autopas/tuning/tuningStrategy/PredictiveTuning.cpp

```diff
@@ -22,7 +22,9 @@ PredictiveTuning::PredictiveTuning(double relativeOptimum, unsigned int maxTunin
       _maxTuningPhasesWithoutTest(maxTuningIterationsWithoutTest),
       _extrapolationMethod(extrapolationMethodOption),
       _minNumberOfEvidence(
-          extrapolationMethodOption == ExtrapolationMethodOption::linePrediction ? 2 : testsUntilFirstPrediction),
+          extrapolationMethodOption == ExtrapolationMethodOption::linePrediction
+              ? 2
+              : (extrapolationMethodOption == ExtrapolationMethodOption::lastResult ? 1 : testsUntilFirstPrediction)),
```

Comment:
```suggestion
      [&]() {
        //The minimal number of evidence needed depends on the extrapolation method
        if (extrapolationMethodOption == ExtrapolationMethodOption::linePrediction) {
          return 2u;
        } else if (extrapolationMethodOption == ExtrapolationMethodOption::lastResult) {
          return 1u;
        } else {
          return testsUntilFirstPrediction;
        }
      }()),
```

Reply:
done in [68558b4](https://github.com/AutoPas/AutoPas/pull/837/commits/68558b40210bf8ae8c1b486f0c1e315b6b1a9029)